WORKING_GROUPS.md

```diff
@@ -192,7 +192,7 @@ Note that changes to WG membership require unanimous consensus.  See
 <a id="developers-certificate-of-origin"></a>
 ## Developer's Certificate of Origin 1.1
 
-*Note*: The DCO is mandatory for all Node.js Foundation projects.
+*Note*: The DCO is mandatory for all OpenJS Foundation projects.
```

Comment:
I don’t think this is accurate - the OpenJS Foundation recommends but does not require a CLA or a DCO.

Reply:
Yeah, let's remove this line entirely.

```suggestion
```

Reply:
Well, that certinly changes things! It's been a talking point in our project (JSON Schema) and maybe others.

It's part of the project progression checklist: https://github.com/openjs-foundation/cross-project-council/blob/main/PROJECT_PROGRESSION.md#onboarding-checklist

Reply:
AFAIK it's a "requirement" in that you can do a DCO or CLA, and that there's leniency if implementing such a process is extremely burdensome (like in the case of Node.js, where we seemingly haven't fully done DCO correctly but are aiming at doing it correctly).

Reply:
It is a requirement and it's documented here: https://github.com/openjs-foundation/cross-project-council/blob/main/IP_POLICY_GUIDANCE.md#4-adopting-the-dco-or-a-cla

Reply:
To be clear the requirement is to choose `either` the DCO or CLA so it's not that a DCO is is required but as @bnb mentioned you need to use one of the two.

Reply:
@mhdawson OK, is the action here modify the wording to say "DCO or CLA"?

Reply:
Resolved by comitting suggested change to this PR